docs/api-testing/sauce-connect.md

```diff
@@ -120,4 +120,5 @@ If you shut down a tunnel that's currently selected in a Sauce Connect dropdown
 
 ## More Information
 
-* [Using Sauce Connect Proxy](/secure-connections/sauce-connect)
\ No newline at end of file
+* [Using Sauce Connect Proxy](/secure-connections/sauce-connect)
+* [Specialized Sauce Connect Setups](/secure-connections/sauce-connect/setup-configuration/specialized-environments/#api-testing-setup)
```

Comment:
Change to "Specialized Sauce Connect Proxy Setups"

Reply:
@lauramthomas good catch! just applied the change -- can you please take a look?